### PR TITLE
Adding pdb2pqr package

### PR DESCRIPTION
**Pull request by tsenapathi**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by tsenapathi** [comment]
@sobolevnrm Here is the pdb2pqr conda package. Can you have a look?

**Comment by tsenapathi** [comment]
@conda-forge-admin, please re-render.

**Review by sobolevnrm — APPROVED** [review]

**Comment by sobolevnrm** [comment]
Looks great; thank you!

File: recipes/pdb2pqr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  license_family: BSD
+  license_file: LICENSE
+  summary: "Automates many of the common tasks of preparing structures for continuum solvation calculations as well as many other types of biomolecular structure modeling, analysis, and simulation."
+  doc_url: https://pdb2pqr.readthedocs.io/en/latest/
+  dev_url: https://github.com/Electrostatics/pdb2pqr
+
+extra:
+  recipe-maintainers:
+    - tsenapathi
+    - sobolevnrm
```
Comment (dopplershift) [inline recipes/pdb2pqr/meta.yaml:64]: @sobolevnrm are you ok being a maintainer of this recipe?

File: recipes/pdb2pqr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    - psize --help
+    - inputgen --help
+  requires:
+    - pytest
+    - requests
+    - testfixtures
+    - setuptools
+
+about:
+  home: "http://www.poissonboltzmann.org"
```
Comment (dopplershift) [inline recipes/pdb2pqr/meta.yaml:53]: Can remove quotes

File: recipes/pdb2pqr/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a6f5e2eefbd8bd2a072a89ee4026977de02dadd56e921095afac88d596b63820
+
+build:
+  number: 0
```
Comment (dopplershift) [inline recipes/pdb2pqr/meta.yaml:13]: Any reason this can't be `noarch: python`? Looks like this package is pure python.

Comment (sobolevnrm) [reply recipes/pdb2pqr/meta.yaml:13]: That's a good point; this is pure Python.  @tsenapathi -- what do you think?

File: recipes/pdb2pqr/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    - inputgen --help
+  requires:
+    - pytest
+    - requests
+    - testfixtures
+    - setuptools
+
+about:
+  home: http://www.poissonboltzmann.org
+  license: BSD
```
Comment (dopplershift) [inline recipes/pdb2pqr/meta.yaml:55]: `BSD-3-Clause`

File: recipes/pdb2pqr/meta.yaml (hunk 3 above)
Comment (tsenapathi) [reply recipes/pdb2pqr/meta.yaml:14]: I agree and changed to` noarch: python`

**Comment by tsenapathi** [comment]
Thanks, @dopplershift for the comments and reviews. 
@sobolevnrm you are fine with adding as a maintainer right?

**Comment by sobolevnrm** [comment]
> Thanks, @dopplershift for the comments and reviews.
> @sobolevnrm you are fine with adding as a maintainer right?

Yes, thank you!

File: recipes/pdb2pqr/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    - dx2cube=pdb2pqr.main:dx_to_cube
+    - psize=pdb2pqr.psize:main
+    - inputgen=pdb2pqr.inputgen:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - mmcif_pdbx >=1.1.2
+    - pandas >=1.0
+    - pip
```
Comment (dopplershift) [inline recipes/pdb2pqr/meta.yaml:26]: Sorry, just noticed this: looks like `host` only needs the `python` and `pip` dependencies?

Comment (tsenapathi) [reply recipes/pdb2pqr/meta.yaml:24]: Oh yes! Thanks for noticing that @dopplershift. I have changed it now.

**Review by dopplershift — APPROVED** [review]

**Comment by tsenapathi** [comment]
Thank you @dopplershift !

**Pull request merged**